src/Compilers/Server/VBCSCompiler/VBCSCompiler.cs

```diff
@@ -13,7 +13,7 @@ internal static class VBCSCompiler
     {
         public static int Main(string[] args)
         {
-            using var logger = new CompilerServerLogger();
+            using var logger = new CompilerServerLogger("VBCSCompiler");
```

Comment:
>"VBCSCompiler" [](http://example.com/codeflow?start=56&length=14)

Perhaps use `nameof(VBCSCompiler)`.

Reply:
If I did that though how would I introduce typos into my PRs?